src/main/kotlin/hu/kirdev/schpincer/service/OrderService.kt

```diff
@@ -100,8 +97,8 @@ open class OrderService {
     @Transactional
     @Throws(IOException::class)
     open fun makeOrder(request: HttpServletRequest, id: Long, itemCount: Int, time: Long, comment: String, detailsJson: String): ResponseEntity<String> {
-        val user = util.getUser(request) ?: return responseOf("Error 403", HttpStatus.FORBIDDEN)
-        if (user.room == null || user.room!!.isEmpty())
+        val user = request.getUserIfPresents() ?: return responseOf("Error 403", HttpStatus.FORBIDDEN)
```

Comment:
Shouldn't this method be called `getUserIfPresent` (without the `s` at the end)?